core/src/main/java/org/apache/iceberg/actions/SizeBasedFileRewriter.java

```diff
@@ -103,13 +103,16 @@
 
   private static final long SPLIT_OVERHEAD = 5 * 1024;
 
+  public static final String OUTPUT_SPEC_ID = "output-spec-id";
```

Comment:
Needs Java doc, see above options like MAX_FILE_GROUP_SIZE
Also this property is probably not SizeBasedFileRewriter Specific. I think it should live in RewriteDataFiles since it applies to all possible future rewrites.

Reply:
Added javadoc.